subsys/bluetooth/host/Kconfig

```diff
@@ -81,6 +81,10 @@ config BT_HCI_TX_STACK_SIZE
 	help
 	  Stack size needed for executing bt_send with specified driver
 
+config BT_HCI_ECC_STACK_SIZE
+	int "HCI ECC thread stack size" if !SOC_COMPATIBLE_NRF
```

Comment:
@jhedberg that `if !` just makes the prompt conditional. So the option is not user configurable for NRF platforms (meaning it is mostly just as before, except that that constant is moved to a hidden kconfig option).
The option only appears and is configurable for other platforms where, due to architecture differences, you may need to tune that stack size.

Reply:
@aescolar so you're saying there are architectures where the same TinyCrypt APIs consume more stack than what's currently configured?  Which architectures are those? If the difference isn't more than 100 bytes or so I'd just increase the global default. If it's more then couldn't we just have architecture-specific defaults for this value?

Reply:
Currently we use 1080 bytes in a test run - this might not be max. We assert when passing 90% usage, just to make sure we have a margin. We would need something like 1200-1300, and _can_ disable the 90% limit for this stack, but it seems close. It seems like an "expensive" solution to generally increase by 200 bytes...